src/poliastro/czml/extract_czml.py

```diff
@@ -129,7 +67,13 @@ def _init_orbit_packet_cords_(self, i):
         ----------
         i: int
             Index of referenced orbit
+
+        Returns
+        -------
+        coordinate list
         """
+        cart_cords = []
```

Comment:
```suggestion
        cart_cords = []  # type: List[?]
```

(not entirely sure what line 85 does)